Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/StabilizationPlaneModifier.cs

```diff
@@ -252,9 +253,11 @@ private Vector3 GazeNormal
         /// <returns>True if gaze is supported and an object was hit by gaze, otherwise false.</returns>
         private bool TryGetGazeHitPosition(out Vector3 hitPosition)
         {
-            if (CoreServices.InputSystem.GazeProvider.Enabled)
+            var gazeProvider = CoreServices.InputSystem?.GazeProvider;
+            if (gazeProvider?.Enabled == true &&
```

Comment:
GazeProvider is currently a MonoBehaviour

```suggestion
            if (gazeProvider != null && gazeProvider.Enabled &&
```

Reply:
oh really? I thought we had migrated it. Good catch

Reply:
Not yet, be we definitely should (I assume you're thinking of https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5938)

Reply:
Very most likely, good call. Excited for that PR.